index.html

```diff
@@ -40,12 +40,169 @@
     </style>
   </head>
   <body>
-    <!--
+    <header>
+      <h2>Travel Blog</h2>
+      <p>Site Name: Travel Blog</p>
+
+      <img 
+      src="https://img.freepik.com/premium-vector/travel-blog-logo-using-pen-plane-as-negative-space_560919-194.jpg"
+      alt="Travel Blog Logo"
+      />
+
+      
+      <nav>
+      <h2>Menu</h2>
+        <ul>
+          <li><a href="#">Home</a></li>
+          <li><a href="#">About</a></li>
+          <li><a href="#">Destinations</a></li>
+          <li><a href="#">Contact</a></li>
+
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+      
+      <h1>Zanzibar, Tanzania</h1>
+
+      <section>
+        <h2>Jambiani Beach</h2>
+      <img
+      src="https://www.easytravel.co.tz/wp-content/uploads/2023/05/Jambiani-in-Zanzibar-Easy-Travel-Tanzania-min-scaled.jpg"
+      alt="Crytal-clear waters and powdery sand"
+      />
+
+      
+
+      <p>
+        From the crystal-clear blue ocean to the powdery sand, it felt as if I was in a painting. Hearing the waves crashing and feeling the warmth of the ocean gave me a sense of peace. Zanzibar is an unforgettable experience I would definently recomend others to visit. From the welcoming warmth of the people to the variety of different sea foods to induldge in.
+       </p>
+       
+       
+       <p>Place's to visit:</p>
+       <ul>
+       <li>Explore Stone Town: Stone town is the heart of Zanzibar with many historical buildings. It's a captivating journey back in time, where every corner holds it's own stories that are waiting to be discovered. </li>
+       
+        <li>Local Cusines: Eating a variety of foods which has a fusion of African, Arabic influences and many more. </li>
+
+        <li>Boat Rides: The crystal-clear waters of the ocean is unmatched and swiming among the fish and possible some dolphins. Feeling the breeze and seeing various types of fish species as you go by gives a sense of peace.  </li>
+      
+      </ul>
+
+      
+
+        <p>
+          Travel Tips
+        </p>
+        <ul>
+        <li>Pack light summer clothes</li>
+        <li>Bring sunscreen</li>
+        <li>Bring Sunglasses</li>
+        <li>Bring Sandles/flipflops</li>
+        <li>Camera to take beautiful pictures</li>
+
+      </ul>
+     
+
+        <p>
+          Scenery Images
+        </p>
```

Comment:
This <p> tag is also more closely resembles a heading then a paragraph of text (so it should be a heading h2/h3)